yarn-project/entrypoints/src/account_entrypoint.ts

```diff
@@ -6,9 +6,51 @@ import { HashedValues, TxContext, TxExecutionRequest } from '@aztec/stdlib/tx';
 
 import { DEFAULT_CHAIN_ID, DEFAULT_VERSION } from './constants.js';
 import { EncodedAppEntrypointCalls } from './encoding.js';
-import type { AuthWitnessProvider, EntrypointInterface, TxExecutionOptions } from './interfaces.js';
+import type { AuthWitnessProvider, EntrypointInterface } from './interfaces.js';
 import { ExecutionPayload } from './payload.js';
 
+/**
+ * The mechanism via which an account contract will pay for a transaction in which it gets invoked.
+ */
+export enum AccountFeePaymentMethodOptions {
+  /**
+   * Signals that some other contract is in charge of paying the fee, nothing needs to be done.
+   */
+  EXTERNAL = 0,
+  /**
+   * Used to make the account contract publicly pay for the transaction with its own fee juice balance,
+   * **which it must already have prior to this transaction**.
+   *
+   * The contract will set itself as the fee payer and end the setup phase.
+   */
+  PREEXISTING_FEE_JUICE = 1,
+  /**
+   * Used to make the account contract publicly pay for the transaction with its own fee juice balance
+   * **which is being claimed in the same tranasction**.
```

Comment:
```suggestion
   * **which is being claimed in the same transaction**.
```

BTW docs now run spellcheck on build so if this was included in docs you would get a CI failure (like me [here](https://aztecprotocol.slack.com/archives/C08HLF3J941/p1759216431721869)).

So recommend installing [the spellcheck extension](https://marketplace.cursorapi.com/items/?itemName=streetsidesoftware.code-spell-checker).

Reply:
Installed!